build/Dockerfile

```diff
@@ -148,12 +149,9 @@ RUN touch /.aim_profile && \
     mkdir /.cache && \
     chmod -R 777 /.cache
 
-# create tuning user and give ownership to dirs
-RUN useradd -u $USER_UID tuning -m -g 0 --system && \
```

Comment:
I moved this line to the [base layer](https://github.com/tharapalanivel/fms-hf-tuning/blob/ee87be95b303cb868af7052b4149c6266a7d41c8/build/Dockerfile#L42) because the [python-installation layer](https://github.com/tharapalanivel/fms-hf-tuning/blob/ee87be95b303cb868af7052b4149c6266a7d41c8/build/Dockerfile#L93-L94) also needs to install python packages in the user site.